### PR TITLE
#161966601 Heroku deploy

### PR DESCRIPTION
**Pull request by verenceLola**
### What does this PR do?
- Configure the application for deployment to Heroku
### Description of the task to be completed?
- Configure the application for hosting on Heroku. 
- Setup the staging and production URLs
### How can this be manually tested?
Try accessing the application on Heroku via [ah-technocrats.herokuapp.com](https://ah-technocrats.herokuapp.com) for `production` and [ah-technocrats-staging.herokuapp.com](https://ah-technocrats-staging.herokuapp.com) for `staging` environment.
### What are the relevant Pivotal Tracker stories?
[#161966601](https://www.pivotaltracker.com/story/show/161966601)

### CODE REVIEW

**Comment by TeamoreA** [comment]
CI checks are failing

**Comment by salma-nyagaka** [comment]
The links seem not to be working, please check on that

**Comment by salma-nyagaka** [comment]
You should have checkpoints after your PRs

**Review by SilasKenneth — DISMISSED** [review]

**Comment by TeamoreA** [comment]
Good for fixing the checks

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Comment by kipropbrian** [comment]
This PR too appears to be corrupted by commits from other branches.

**Comment by verenceLola** [comment]
@kipropbrian the conflicts have been resolved.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```
Comment (Mnickii) [inline README.md]: This is unrelated to the deploy task.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+## Prerequisite
+- [Python3.6](https://www.python.org/downloads/release/python-365/)
+- [Virtual Environment](https://virtualenv.pypa.io/en/stable/installation/)
+
+## Installation and Setup
+
+Clone the repository below
+
+    -git clone https://github.com/andela/ah-technocrats.git
+    -git checkout develop
```
Comment (Mnickii) [inline README.md]: This too

File: authors/apps/authentication/tests/test_login.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            self.register_data,
+            format="json"
+        )
+        response = self.client.post(
+            self.login_url,
+            self.login_data,
+            format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_login_with_no_data(self):
```
Comment (Mnickii) [inline authors/apps/authentication/tests/test_login.py]: Neither is this

**Review by Mnickii — CHANGES_REQUESTED** [review]
Make sure only your changes pertaining to your story are present in this PR

File: README.md (hunk 1 above)
Comment (verenceLola) [reply README.md]: Thanks for your review @Mnickii. I have resolved the issue.

File: README.md (hunk 2 above)
Comment (verenceLola) [reply README.md]: I have resolved this too.

**Review by Mnickii — APPROVED** [review]
